stable/drupal/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: drupal
-version: 0.4.6
+version: 0.4.7
```

Comment:
On second thought, since the mariadb upgrade is non-intrusive to the drupal charts upgrades, i would prefer staying on the existing minor version series /cc @prydonius